Library/Homebrew/cmd/formulae.sh

```diff
@@ -0,0 +1,13 @@
+#:  * `formulae`
+#:
+#:  Prints a sorted list of locally available formulas including shortnames.
+#:
+
+# shellcheck disable=SC2155
+homebrew-formulae() {
+  local formulae="$(
+    find "$HOMEBREW_REPOSITORY"/Library/Taps -type d \( -name Casks -o -name cmd -o -name .github \) -prune -false -o -name '*\.rb' 
+      | sed -E -e 's/\.rb//g' -e 's .*/Taps/(.*)/homebrew-\1/ g' -e 's /Formula/ / g')"
```

Comment:
do we need `-E` here?
`'s .*/Taps/(.*)/homebrew-\1/ g'` seems malformed. Did you mean `-e 's|.*/Taps/\(.*\)|homebrew-\1|g'`? (`\(` and `\)` instead of `(` and `)` without `-E`.

I'd suggest to `|` symbol for splitting patterns because it let's us avoid escaping `/` symbol in the second substitution.

Reply:
Can't use that commit as-is, or it will fail one of shellcheck's tests.  

Apparently it has strong opinions on where the `|` goes.  Sorry, I didn't see that you had made a suggestion before I committed the changes, but its essentially your commit but with the | moved so shellcheck doesn't complain.  

@MikeMcQuaid I also added a couple more filters to the find command, since the recent inclusions of the new homebrew-bundler and several other new folders inside the homebrew/homebrew-core tap folder were getting picked up by the search.  Fortunately, adding directory names to skip has virtually no impact on the speed.  I wish I could limit the search to the Formula directory, but that would break backwards compatibility with older taps that just have the formula in the tap folder.